citizen/citizen.c

```diff
@@ -401,33 +446,35 @@ int getMalePopulation(List *list) {
  * @param list A pointer to the list of citizens.
  * @return The number of female citizens in the list.
  */
-int getFemalePopulation(List *list) {
-    Node *current = list->head;   // Start from the head of the list
-    int femalePopulation = 0;     // Counter for female population
+int getfemale_population(List *list)
```

Comment:
please me consistent, the previous uses camelCase on the function naming convention

Reply:
Renamed. Thanks.